Edited ML translations (#48)
IllusionMods/KKS-Translation: Translation/en/RedirectedResources/assets/abdata/communication/info_00/tips_00/translation.txt

//
// Dumped for Koikatsu Sunshine v1.0.2 (KoikatsuSunshine) by KKS_TextDump v1.4.5.1
//

> ヤマネコ >=Wildcat

Reviewer: I think we want to keep the tip formatting as is, so put ` >` at the end.
```suggestion
ヤマネコ >=Wildcat >
```

> <color%3D#ffa801ff>ヤマネコ</color>を<color%3D#ffa801ff>自然公園</color>や<color%3D#ffa801ff>路地</color>でたまーに見かけるって聞くよ。見つけられたらラッキー！=I hear that <color%3D#ffa801ff>wildcats</color> are sometimes seen in <color%3D#ffa801ff>nature parks</color> and <color%3D#ffa801ff>alleyways</color>. You're lucky if you can find one!

Reviewer: ```suggestion
<color%3D#ffa801ff>ヤマネコ</color>を<color%3D#ffa801ff>自然公園</color>や<color%3D#ffa801ff>路地</color>でたまーに見かけるって聞くよ。見つけられたらラッキー！=I hear that <color%3D#ffa801ff>wildcats</color> are sometimes seen in <color%3D#ffa801ff>nature parks</color> and <color%3D#ffa801ff>alleyways</color>. If you're lucky, you might see one!

> ヤシガニ >=Coconut Crabs

Reviewer: ```suggestion
ヤシガニ >=Coconut Crabs >
```

> <color%3D#ffa801ff>浜辺</color>や<color%3D#ffa801ff>自然公園</color>には稀に<color%3D#ffa801ff>ヤシガニ</color>が出る事があるよ。立派なハサミで挟まれないように注意！=In <color%3D#ffa801ff>beaches</color> and <color%3D#ffa801ff>nature parks</color>, <color%3D#ffa801ff>coconut crabs</color> rarely appear. Be careful not to get pinched by their magnificent claws!

Reviewer 2: ```suggestion
<color%3D#ffa801ff>浜辺</color>や<color%3D#ffa801ff>自然公園</color>には稀に<color%3D#ffa801ff>ヤシガニ</color>が出る事があるよ。立派なハサミで挟まれないように注意！=In <color%3D#ffa801ff>beaches</color> and <color%3D#ffa801ff>nature parks</color>, <color%3D#ffa801ff>coconut crabs</color> rarely appear. Be careful not to get pinched by their magnificent pincers!
```
Pincers are the correct term for crabs.

Author: Correct! I saw claws as better suited for the non-formal nature of the statement.

Reviewer 2: Sounds fair, if no one else objects then "claws" should be fine.

> ロマンチックな告白は…… >=Confessions

Reviewer: Maybe something like:
```suggestion
ロマンチックな告白は…… >=Need a romantic spot to confess...
```

Reviewer 3: Once this is resolved the PR looks fine to merge to me

Author: > Maybe something like:

How about `Need a Romantic Spot to Confess?` since it's the title for the tip

Author: Updated to "Need a Romantic Spot to Confess?"

> <color%3D#ffa801ff>流れ星が綺麗に見える</color>ような静かな場所で愛の<color%3D#ffa801ff>告白</color>……ロマンチックな告白に憧れる～！=I long for a <color%3D#ffa801ff>romantic confession of love</color> in a quiet place where I can <color%3D#ffa801ff>see shooting stars</color>......

Reviewer 2: ```suggestion
<color%3D#ffa801ff>流れ星が綺麗に見える</color>ような静かな場所で愛の<color%3D#ffa801ff>告白</color>……ロマンチックな告白に憧れる～！=Every girl long for a <color%3D#ffa801ff>romantic love confession</color>... like one in a quiet place where you can <color%3D#ffa801ff>see shooting stars</color>.
```

Author: Every girl **longs** for  a romantic <color%3D#ffa801ff>confession of love</color> ... like one in a quiet place where you can <color%3D#ffa801ff>see shooting stars</color>.

- How about this? Combines both, centers the highlight on confession mechanic.

Reviewer: `girl long` => `girl longs`

Author: Commited changes

> もう一つのビーチ？ >=Another Beach?

Reviewer: ```suggestion
もう一つのビーチ？ >=Another Beach? >
```

> この島には立ち入り禁止になってる<color%3D#ffa801ff>もう一つのビーチ</color>があるんだよ。今はどうなってるかなぁ～？= There's <color%3D#ffa801ff>another beach</color> on the island that's off-limits. I wonder what's happening there right now~?

Reviewer 2: ```suggestion
この島には立ち入り禁止になってる<color%3D#ffa801ff>もう一つのビーチ</color>があるんだよ。今はどうなってるかなぁ～？= There's <color%3D#ffa801ff>another beach</color> on the island that's off-limits. I wonder what's it's like there now~?
```
Question is about the state of the beach after all this time (what's changed/if it looks different), not what's happening at this moment.

> ウムイの鐘 >=The Bell of Umui 

Reviewer: in `Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt` this is `Umui Bell`. They should match.  I originally had `The Umui Bell`.  I think `The Umui Bell` or `The Bell of Umui` is fine

Author: Is this good to go?

Reviewer 2: LGTM

> //ウムイの鐘 >=The Bell of Umui 

Reviewer 2: ```suggestion
ウムイの鐘 >=The Bell of Umui 
```
Umui is a name, should be fine to be left untranslated.
https://kotobank.jp/word/%E3%82%A6%E3%83%A0%E3%82%A4-819936

> //<color%3D#ffa801ff>灯台</color>の近くに<color%3D#ffa801ff>ウムイの鐘</color>がおいてあるよ。<color%3D#ffa801ff>恋人同士</color>でその鐘を鳴らすと、その二人は必ず結ばれるっていう噂があるんだ～。=Near the <color%3D#ffa801ff>lighthouse</color>, you'll find the <color%3D#ffa801ff>Bell of Umui</color>. There's a rumor that if two <color%3D#ffa801ff>lovers</color> ring the bell, they're sure to get married.

Reviewer 2: ```suggestion
<color%3D#ffa801ff>灯台</color>の近くに<color%3D#ffa801ff>ウムイの鐘</color>がおいてあるよ。<color%3D#ffa801ff>恋人同士</color>でその鐘を鳴らすと、その二人は必ず結ばれるっていう噂があるんだ～。=Near the <color%3D#ffa801ff>lighthouse</color>, you'll find the <color%3D#ffa801ff>Bell of Umui</color>. There's a rumor that if two <color%3D#ffa801ff>lovers</color> ring the bell, they're sure to get married.
```

カップルドリンク >=Couple Drink

> <color%3D#ffa801ff>ホテルのプールサイド</color>で注文できる<color%3D#ffa801ff>トロピカルドリンク</color>が人気だよ。<color%3D#ffa801ff>恋人同士</color>で一緒に飲むのがブームみたい。=The <color%3D#ffa801ff>tropical drink</color> that you can order at the <color%3D#ffa801ff>hotel poolside</color> is very popular. <color%3D#ffa801ff>Lovers</color> like to drink it together.

Reviewer 2: ```suggestion
<color%3D#ffa801ff>ホテルのプールサイド</color>で注文できる<color%3D#ffa801ff>トロピカルドリンク</color>が人気だよ。<color%3D#ffa801ff>恋人同士</color>で一緒に飲むのがブームみたい。=The <color%3D#ffa801ff>tropical drink</color> that you can order at the <color%3D#ffa801ff>hotel poolside</color> is very popular. There's currently a fad where <color%3D#ffa801ff>couples</color> drink it together.
```

> デートスポット >=Date Spots

Reviewer: ```suggestion
デートスポット >=Date Spots >
```

この島には<color%3D#ffa801ff>カップル</color>で楽しめる<color%3D#ffa801ff>スポット</color>がいくつかあるんだよ。そういう相手がいるなら一緒に周ってみてね！=There are a few <color%3D#ffa801ff>couple spots</color> on this island. If you have a partner, you should go visit them together!

> バナナボート >=Banana Boat

Reviewer: ```suggestion
バナナボート >=Banana Boat >
```

> <color%3D#ffa801ff>浜辺のレンタルショップ</color>で頼めば<color%3D#ffa801ff>バナナボート</color>を体験できるよ。よかったら<color%3D#ffa801ff>恋人同士</color>で一緒に体験してみてね！=If you ask at the <color%3D#ffa801ff>rental shop</color> on the beach, you can try a <color%3D#ffa801ff>banana boat</color>. If you'd like, <color%3D#ffa801ff>lovers</color> can experience it together!

Reviewer 2: ```suggestion
<color%3D#ffa801ff>浜辺のレンタルショップ</color>で頼めば<color%3D#ffa801ff>バナナボート</color>を体験できるよ。よかったら<color%3D#ffa801ff>恋人同士</color>で一緒に体験してみてね！=If you ask at the <color%3D#ffa801ff>rental shop</color> on the beach, you can try out a <color%3D#ffa801ff>banana boat</color>. If that piques your interest, bring your <color%3D#ffa801ff>lover</color>along and experience it together!
```
